Python/Include/TrialView.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef ROVER_PYTHON_TRIAL_VIEW_HPP
+#define ROVER_PYTHON_TRIAL_VIEW_HPP
+#include <string_view>
+#include <type_traits>
+#include <pybind11/pybind11.h>
+#include "Rover/ListTrial.hpp"
+#include "Rover/TrialView.hpp"
+#include "Sample.hpp"
```

Comment:
Include it via "Rover/Python/Sample.hpp" otherwise third party libraries won't be able to use this header file to export their own custom instantiations of ```TrialView```.

Reply:
I don't really have a canonical include path for Rover/Python. Include files reside directly in Rover/Python/Include/*.hpp, I think you put a few files there initially and I just kept it up. Should I move those to Rover/Python/Include/Rover/Python? If so, I think it is better done as a separate case so that it does not interfere with the changes here.

Reply:
Okay, add a case for canonical paths.

Reply:
Add the case in Fogbugz and then mark this as resolved.